### DISCUSSION TITLE
Is there any way to set the friction torque to a joint ?

### DISCUSSION THREAD

**Discussion by momadacoding** [NVIDIA-Omniverse/PhysX#68] [Q&A] [answered]
Does driveDamping in PxD6JointDrive do the same thing?

**Comment by amoravanszky** [comment, marked as answer]
Yes, the damping term will drive toward the specified targetVelocity, so if you set targetVelocity to zero it should behave like friction.

If you need to simultaneously actuate the joint to reach a nonzero target velocity, you can model frictional losses in the joint by making the damping term be lower than it would be for a frictionless joint.
